.changeset/long-buckets-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': major
```

Comment:
this would have to be changed to `minor`.

Reply:
Sorry, that was a typo. Fixed now.